WTWSMS/decisions/GR_Scythian.txt

```diff
@@ -203,7 +203,7 @@ decisions = {
 	}
 }
 
-targetted_decisions = {
+decisions = {
```

Comment:
Your sole reason for being opposed to targeted decisions is that they are invisible. Yet that is something we can configure with the priority definition.